core/include/tee/tee_fs.h

```diff
@@ -58,6 +58,8 @@ extern const struct tee_file_operations rpmb_fs_ops;
 
 TEE_Result tee_rpmb_fs_raw_open(const char *fname, bool create,
 				struct tee_file_handle **fh);
+
+bool plat_rpmb_ready(void);
```

Comment:
I'd prefer `plat_rpmb_key_is_ready(void)`.
Please add a short description of the function.